### ISSUE TITLE
First Studies 

### ISSUE THREAD

**Issue by kratsg** [kratsg/TheAccountant#61]
- [ ] Find a top-tagged jet in data, and tell me everything you can about it.
- [ ] Show the output distributions (meff, ht, met_sig) as a function of the number of top tags, in the ttbar sample

**Comment by kratsg** [comment]
File: `data15_13TeV.00266919.physics_Main`

```
1111|run#266919|event#12571271|lumi#229
10-1-1|run#266919|event#12827550|lumi#233
11-1-1|run#266919|event#17437041|lumi#295
0010|run#266919|event#5836781|lumi#144
10-1-1|run#266919|event#10315690|lumi#199
10-1-1|run#266919|event#11442677|lumi#214
10-1-1|run#266919|event#15179186|lumi#263
```

Top Tagging order (1234)
1. SmoothLoose
2. SmoothTight
3. Loose
4. Tight

**Comment by mswiatlo** [comment]
bump (more priority on the second point above now)

**Comment by kratsg** [comment]
#66 is related: 

Ok, I'm more or less done with hardware. Need to get home. Get food. Clean. Be all adult. I'll spin up jobs tonight that will be ready in the morning... and I need to think hard about the plotting because it's probably easier to avoid 2D histograms, and just figure out how to add the 1D histograms of everything we want. 

... right? my first thought is that I add a subset where we have.. `jetsLargeR/decorations/TopTagLoose` or something and within that is the same standard set of kinematic and multiplicity plots.

**Comment by mswiatlo** [comment]
Sounds good. (Also keep in mind when I dump issues on here, I don’t mean for them to be done immediately immediately, but just want to keep a list of what our goals are for the semi-short term).

I like that idea for organization. You can do something like TopTagLoose1, TopTagLoose2, etc., which require >=1, >=2, etc. tags.

**Comment by kratsg** [comment]
How would one differentiate >= 1 top tag versus ==1 top tag? Maybe 'geq1' and 'eq1' prefixed.

**Comment by mswiatlo** [comment]
Yeah something like that is fine to keep things general, incase we want to do other things besides >=.
